[DOCS] Final review for 2.0.0

frascuchon opened the pull request:
# Description
<!-- Please include a summary of the changes and the related issue. Please also include relevant motivation and context. List any dependencies that are required for this change. -->


**Type of change**
<!--  Please delete options that are not relevant. Remember to title the PR according to the type of change  -->

- Documentation update

**How Has This Been Tested**
<!--  Please add some reference about how your feature has been tested.  -->

**Checklist**
<!--  Please go over the list and make sure you've taken everything into account -->

- I added relevant documentation
- I followed the style guidelines of this project
- I did a self-review of my code
- I made corresponding changes to the documentation
- I confirm My changes generate no new warnings
- I have added tests that prove my fix is effective or that my feature works
- I have added relevant notes to the CHANGELOG.md file (See https://keepachangelog.com/)

dvsrepo committed:
fix heading

frascuchon committed:
fix: Link to the server telemetry module

davidberenstein1957 committed:
Add custom subdomain (#5328)

# Description
<!-- Please include a summary of the changes and the related issue.
Please also include relevant motivation and context. List any
dependencies that are required for this change. -->

Closes #5327 

**Type of change**
<!-- Please delete options that are not relevant. Remember to title the
PR according to the type of change -->

- Documentation update

**How Has This Been Tested**
<!-- Please add some reference about how your feature has been tested.
-->

**Checklist**
<!-- Please go over the list and make sure you've taken everything into
account -->

- I added relevant documentation
- I followed the style guidelines of this project
- I did a self-review of my code
- I made corresponding changes to the documentation
- I confirm My changes generate no new warnings
- I have added tests that prove my fix is effective or that my feature
works
- I have added relevant notes to the CHANGELOG.md file (See
https://keepachangelog.com/)

---------

Co-authored-by: Paco Aranda <frascuchon@gmail.com>
Co-authored-by: Leire <leire@argilla.io>

frascuchon committed:
Merge branch 'releases/2.0.0' into docs/2.0.0

sdiazlor committed:
Update tutorials docker reference

sdiazlor committed:
fix quickstart link and duplicated subheader

jfcalvo committed:
docs: remove some references to quickstart docker image

frascuchon commented on argilla/docs/how_to_guides/workspace.md:
```diff
@@ -18,9 +18,9 @@ A **workspace** is a *space* inside your Argilla instance where authorized users
 |-------------------------------|----------|
-| Quickstart Docker and HF Space | admin    |
+| Docker and HF Spaces          | admin    |
 | Server image                  | argilla  |
```
This section does not apply anymore. @sdiazlor did you already change this on https://github.com/argilla-io/argilla/pull/5337?

sdiazlor replied:
No, not yet. @frascuchon so is there no default workspace either?

frascuchon replied:
No. For HF spaces the one configured in `.oauth.yaml`. For common docker deployment, the owner must create a workspace first.

frascuchon replied:
So, maybe replacing this table with the different scenarios depending on the deployment makes sense.

sdiazlor committed:
tutorials reference to quickstart

frascuchon committed:
Merge branch 'releases/2.0.0' into docs/2.0.0

sdiazlor committed:
docs: How-to guides review (#5337)

# Description
<!-- Please include a summary of the changes and the related issue.
Please also include relevant motivation and context. List any
dependencies that are required for this change. -->

Closes #5332 

**Type of change**
<!-- Please delete options that are not relevant. Remember to title the
PR according to the type of change -->

- Bug fix (non-breaking change which fixes an issue)
- New feature (non-breaking change which adds functionality)
- Breaking change (fix or feature that would cause existing
functionality to not work as expected)
- Refactor (change restructuring the codebase without changing
functionality)
- Improvement (change adding some improvement to an existing
functionality)
- Documentation update

**How Has This Been Tested**
<!-- Please add some reference about how your feature has been tested.
-->

**Checklist**
<!-- Please go over the list and make sure you've taken everything into
account -->

- I added relevant documentation
- I followed the style guidelines of this project
- I did a self-review of my code
- I made corresponding changes to the documentation
- I confirm My changes generate no new warnings
- I have added tests that prove my fix is effective or that my feature
works
- I have added relevant notes to the CHANGELOG.md file (See
https://keepachangelog.com/)

---------

Co-authored-by: Daniel Vila Suero <daniel.vila@huggingface.co>
Co-authored-by: pre-commit-ci[bot] <66853113+pre-commit-ci[bot]@users.noreply.github.com>
Co-authored-by: Paco Aranda <frascuchon@gmail.com>

dvsrepo committed:
tiny improvement

sdiazlor committed:
fix small typos

dvsrepo committed:
Merge branch 'releases/2.0.0' into docs/2.0.0

dvsrepo commented on README.md:
```diff
@@ -34,7 +34,7 @@
 Argilla is a collaboration tool for AI engineers and domain experts who need to build high-quality datasets for their projects.
 
-If you just want to get started, we recommend our [UI demo](https://demo.argilla.io/sign-in?auth=ZGVtbzoxMjM0NTY3OA%3D%3D) or our [free Hugging Face Spaces deployment integration](https://huggingface.co/new-space?template=argilla/argilla-template-space). Curious, and want to know more? Read our [documentation](https://argilla-io.github.io/argilla/latest/).
+If you just want to get started, we recommend our [UI demo](https://demo.argilla.io/sign-in?auth=ZGVtbzoxMjM0NTY3OA%3D%3D) or our [free Hugging Face Spaces deployment integration](https://huggingface.co/new-space?template=argilla/argilla-template-space). Curious, and want to know more? Read our [documentation](https://docs.argilla.io/latest/).
```
```suggestion
If you just want to get started, [deploy Argilla on Hugging Face Spaces](https://docs.v2.argilla.io/latest/getting_started/quickstart/). Curious, and want to know more? Read our [documentation](https://docs.v2.argilla.io/latest/).
```

dvsrepo committed:
simplify quickstart
